src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.component.ts

```diff
@@ -130,19 +130,25 @@ export class OnlineGameWrapperComponent extends GameWrapper implements OnInit, O
     private isOpponent(player: PlayerOrNone): boolean {
         return this.observerRole !== player.value;
     }
-    private async redirectIfPartIsInvalid(): Promise<void> {
-        const gameType: string = this.extractGameTypeFromURL();
-        const partValidity: MGPValidation =
-            await this.gameService.getPartValidity(this.currentPartId, gameType);
-        if (partValidity.isFailure()) {
-            // note, option if WRONG_GAME_TYPE to redirect to another page
+    private async redirectIfPartOrGameIsInvalid(): Promise<void> {
+        const gameName: string = this.extractGameNameFromURL();
+        const gameExists: boolean = GameInfo.ALL_GAMES().some((gameInfo: GameInfo) => gameInfo.urlName === gameName);
+        if (gameExists === false) {
```

Comment:
prefer if true else to if false else !

Reply:
J'PÈTE LE FEU CE MATIN je recommence à demander des trucs un peu trop du domaine du détail :D ! ça te va :D ?